managers/bosh-manager/DirectorService.js

```diff
@@ -232,22 +231,14 @@ class DirectorService extends BaseDirectorService {
     const operation = {
       type: 'update'
     };
-    // service fabrik operation token
-    const token = _.get(params.parameters, 'service-fabrik-operation', null);
-    if (token) {
-      _.unset(params.parameters, 'service-fabrik-operation');
-      _.set(params, 'scheduled', true);
-    }
     return this
       .initialize(operation)
-      .then(() => token ? jwt.verify(token, config.password) : null)
-      .then(serviceFabrikOperation => {
-        logger.info('SF Operation input:', serviceFabrikOperation);
-        this.operation = _.get(serviceFabrikOperation, 'name', 'update');
+      .then(() => {
+        logger.info('Parameters for update operation:', _.get(params, 'parameters'));
         // normal update operation
```

Comment:
Please remove this comment now. I should have only removed it in my PR.

Reply:
done